sfm/ui/views.py

```diff
@@ -521,10 +521,12 @@ def _get_fileinfos(path):
     if os.path.exists(path):
         contents = os.listdir(path)
         for item in contents:
-            path = os.path.join(path, item)
-            if os.path.isfile(path):
-                fileinfos.append((item, os.path.getsize(path)))
-    return fileinfos
+            p = os.path.join(path, item)
+            if os.path.isfile(p):
+                fileinfos.append((item, os.path.getsize(p)))
+    # sorted the list based the int from the file suffix, t1,t2,t10
```

Comment:
This can be simplified once padding is used for the suffix.  See comments in sfm-utils.